samples/apps/autogen-studio/autogenstudio/workflowmanager.py

```diff
@@ -931,7 +1036,23 @@ def receive(
         silent: Optional[bool] = False,
     ):
         if self.message_processor:
-            self.message_processor(sender, self, message, request_reply, silent, sender_type="agent")
+            if isinstance(message, dict) and message.get("tool_calls"):
+                tool_calls = [
+                    {
+                        "name": func["name"],
+                        "arguments": ', '.join(f'{key}: {value}' for key, value in json.loads(func["arguments"]).items())
```

Comment:
This causes some issues as it is.  Arguments can sometimes be super long.  I've got a fix coming tomorrow.